modules/fedimint-ln-client/src/lib.rs

```diff
@@ -903,7 +904,8 @@ impl LightningClientModule {
             .get_value(&MetaOverridesKey {})
             .await
         {
-            if meta.fetched_at.elapsed().unwrap() < META_OVERRIDE_CACHE_DURATION {
+            let elapsed = now().duration_since(meta.fetched_at).unwrap();
```

Comment:
BTW. We should have probably write a either a free-standing `elapsed` function in `fedimint_core` (just like `now`), or even use an extension trait.

Reply:
Good point 👍 

Tracking in https://github.com/fedimint/fedimint/issues/4383